### DISCUSSION TITLE
Daily update script?

### DISCUSSION THREAD

**Discussion by nmathey** [MadeInPierre/finalynx#116] [Q&A] [answered]
Salut,

Peux-tu partager l'exemple de ton script que tu lances quotidiennement?
Je cherche par exemple à forcer le fetch de Finary (-f) sans avoir à confirmer que je souhaites réutiliser les identifiants déjà enregistrés

**Comment by MadeInPierre** [comment]
Hello! J'hésite à publier ma config perso (longue et probablement assez complète même si elle n'utilise pas toutes les features), ça pourrait être utile de la mettre dans `finalynx/examples/author_config.py` tu penses ? Un peu personnel comme infos quand même :sweat_smile: 

Quel est le souci avec l'option `-f` ? Tu l'utilises souvent (normalement pas besoin) ? Le fait que Finalynx pose des questions dans le terminal est problématique ?

**Reply by nmathey** [reply]
Non je pensais que tu avais un script qui se lançait tout seul automatiquement (avec l'option -f je suppose) et que si c'est auto alors pas besoin de confirmer l'utilisation des crédentials locaux.
Mais j'ai peut être mal compris ce que tu avais dit ^^

**Reply by MadeInPierre** [reply, marked as answer]
Ah! J'ai juste un petit script linux qui démarre simplement `python assistant_config.py` quand mon ordi démarre, pas besoin de `-f` (qui efface le cache et les cookies qui contiennent la session sauvegardée) ni de `-c` (efface le cache mais laisse les cookies).

Optionnellement, tu peux utiliser `python assistant_config.py -c` pour forcer Finalynx à supprimer le cache et aller chercher les valeurs dans Finary (et RealT aussi maintenant). Par défaut, le cache est valide pendant 12h et Finalynx réutilise le cache tant qu'il est valide.

`-f` n'a effet que pour la source Finary car RealT n'a pas de credentials.
